site/content/how-to/maintenance/upgrade-apps-without-downtime.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Upgrade applications without downtime"
+description: "Learn how to use NGINX Gateway Fabric to upgrade applications without downtime."
```

Comment:
this guide suites better in traffic management section - because it is still about configuring traffic rules to prevent downtime during application upgrades.

I'd expect 'Maintenance and Upgrades' section will include content about maintaining and upgrading NGF itself, rather than apps.

Reply:
This is outside the current scope of this PR. We want to merge this as soon as possible, which will make further updates to the docs easier and avoid having to update two branches in parallel. Let's continue the conversation about further IA changes in Slack.